app/models/rental.rb

```diff
@@ -95,7 +95,14 @@ def renter_is_assignable
   def create_reservation
     throw :abort unless valid? # check if the current rental object is valid or not
     begin
-      reservation = Inventory.create_reservation(item_type.name, start_time, end_time)
+
+      # Because integration testing using development environment (not fixable
```

Comment:
Hmm. Do you mean that the `RAILS_ENV == 'development'`? That doesn't seem right.

Reply:
Feature testing requires an actively running server, which in turn causes the tests to run off the development database. This has been documented in many things selenium-related. I assume it's not fixable because I don't remember a case that *didn't* use the development database then just use DatabaseCleaner to rollback the database after the tests.

Reply:
For what it's worth, I have tried to manually set the environment during the tests and change it back after, which didn't work.

Reply:
Ah okay, interesting! I guess in most scenarios (where their development database doesn't make any outside calls) it's fine for people.